src/dispatch/alembic/versions/5d4dee3e24fc_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
```

Comment:
nit: add a message (e.g. adds column for conference challenge)

Reply:
Oops, good catch.